service/role.go

```diff
@@ -212,7 +212,7 @@ func (r *roleService) StartRoleUpdater(ctx context.Context) <-chan error {
 		}
 	}()
 
-	r.domainRoleCache.EnableExpiredHook().SetExpiredHook(r.handleExpiredHook).StartExpired(ctx, expiryHookRefreshInterval)
+	r.domainRoleCache.StartExpired(ctx, expiryCheckInterval)
```

Comment:
```suggestion
	r.domainRoleCache.EnableExpiredHook().SetExpiredHook(func (ctx context.Context, key string) {
	glg.Warnf("handleExpiredHook(), key: %s", key)
}).StartExpired(ctx, expiryCheckInterval)
```
something like this? use inline